src/main/java/com/amazon/janusgraph/schema/TravelSchemaBuilder.java

```diff
@@ -0,0 +1,110 @@
+package com.amazon.janusgraph.schema;
```

Comment:
also, does this schema have anything to do with your travel graph? seems like it is totally unrelated as the entities in your travel graph are hotels. maybe you should remove this class